src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/dialogs/projectManipulation/pages/JavaProjectThirdPage.java

```diff
@@ -0,0 +1,153 @@
+        textField("Module name:", true).setText(moduleName);
+    }
+
+    /**
+     * Get the location of the content root currently inserted in the 'Content root' input field
+     *
+     * @return location of the content root currently inserted in the input field
+     */
+    public String getContentRoot() {
+        return textField("Content root:", true).getText();
+    }
+
+    /**
+     * Insert the location of the content root into the 'Content root' input field
+     *
+     * @param contentRoot location of the content root that will be set into the input field
+     */
+    public void setContentRoot(String contentRoot) {
+        textField("Content root:", true).setText(contentRoot);
+    }
+
+    /**
+     * Get the location of the module file currently inserted in the 'Module file location' input field
+     *
+     * @return location of the module file currently inserted in the input field
+     */
+    public String getModuleFileLocation() {
+        return textField("Module file location:", true).getText();
+    }
+
+    /**
+     * Insert the location of the module file into the 'Module file location' input field
+     *
+     * @param moduleFileLocation location of the module file that will be set into the input field
+     */
+    public void setModuleFileLocation(String moduleFileLocation) {
+        textField("Module file location:", true).setText(moduleFileLocation);
+    }
+
+    /**
+     * Get the project format currently set in the 'Project format' combo box
+     *
+     * @return project format currently set in the combo box
+     * @throws UITestException when there is set another value than defined by the 'ProjectFormatType' enumeration in the combo box
+     */
+    public ProjectFormatType getProjectFormat() {
+        ComboBoxFixture projectFormatComboBox = comboBox(byXpath("//div[@class='JComboBox']"), Duration.ofSeconds(10));
+
+        if (projectFormatComboBox.selectedText().contains(ProjectFormatType.IDEA_DIRECTORY_BASED.toString())) {
+            return ProjectFormatType.IDEA_DIRECTORY_BASED;
+        } else if (projectFormatComboBox.selectedText().contains(ProjectFormatType.IPR_FILE_BASED.toString())) {
+            return ProjectFormatType.IPR_FILE_BASED;
+        } else {
+            throw new UITestException("Currently selected project format is not supported.");
+        }
+    }
+
+    /**
+     * Set the project format into the 'Project format' combo box
+     *
+     * @param projectFormatType project format that will be set into the combo box
+     */
+    public void setProjectFormat(ProjectFormatType projectFormatType) {
+        ComboBoxFixture projectFormatComboBox = comboBox(byXpath("//div[@class='JComboBox']"), Duration.ofSeconds(10));
+        projectFormatComboBox.selectItemContains(projectFormatType.toString());
+    }
+
+    /**
+     * Enumeration defining values of the 'Project format' combo box
+     */
+    public enum ProjectFormatType {
```

Comment:
Can you please extract the enum to the appropriate package? It might be of use in another places.